src/Culling/babylon.boundingBox.ts

```diff
@@ -131,37 +141,50 @@
          * @returns current bounding box
          */
         public setWorldMatrix(matrix: Matrix): BoundingBox {
-            this._worldMatrix.copyFrom(matrix);
+            this._worldMatrix = matrix;

```

Comment:
Do not modified in place the `_worldMatrix`, for 2 reasons :
- this World matrix can be the one provided by the geometry, which should not be modified from here
- we don't want to modify `_identityMatrix`